src/schema/schema.ts

```diff
@@ -368,11 +368,21 @@ export function createZimbraSchema(options: ZimbraSchemaOptions): {
 						message as SendMessageInput,
 						accountName as string
 					),
-				sendMessage: (_, { message, accountName }, context = {}) =>
-					(context.local ? localStoreClient : client).sendMessage(
-						message as SendMessageInput,
-						accountName as string
-					),
+				sendMessage: (_, { message, accountName, sign = false, encrypt = false }, context = {}) => {
```

Comment:
these changes will not be required if we do https://github.com/Zimbra/zm-api-js-client/pull/481/files#r704185616

Reply:
Fixed.